charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -64,6 +64,10 @@ spec:
             secretKeyRef:
               name: {{ template "common.naming.fullname" . }}
               key: personalAPIKey
+        {{- with (include "common.proxy" .) }}
```

Comment:
We can also add unit test. (not really needed, just mentioning in case you want to check how those works)

Reply:
Ok, I am going to finish the e2e first and then come back to this, thanks for pointing out.

Reply:
Added here 70a85e6ab82eeac56fd0f9cb4278044deee32e04

Reply:
thanks for adding them!